packages/builder/src/stores/backend/tables.js

```diff
@@ -90,6 +90,23 @@ export function createTablesStore() {
     await fetch()
   }
 
+  const assignDisplayColumn = ({
```

Comment:
Now that we have `vitest`, I think it would be good to see a unit test for this.

Reply:
Unit tests added for the tables store